assets/js/messages.json

```diff
@@ -120,9 +128,9 @@
         ],
         [
             {
-                "project": ["mc", "mcd", "mcpe", "mcl", "mce"],
+                "project": ["mc", "mcd", "mcpe", "mcl", "mce", "realms"],
                 "name": "Duplicate",
-                "message": "*Thank you for your report!*\nWe're actually already tracking this issue in *%s%*, so I've resolved and linked this ticket as a duplicate.\n\nIf you would like to add a vote and any extra information to the main ticket it would be appreciated.\n\nIf you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=%project_id%] to see if the issue has already been mentioned.\n\n%quick_links%",
+                "message": "*Thank you for your report!*\nWe're tracking this issue as *%s%*, so this ticket is being resolved and linked as a *duplicate*.\n\nIf you would like to add a vote and any extra information to the main ticket it would be appreciated.\n\nIf you haven't already, you might like to make use of the [*+search feature+*|https://bugs.mojang.com/issues/?jql=project=%project_id%] to see if the issue has already been reported.\n\n%quick_links%",
```

Comment:
In this message you replaced `mentioned` with `reported`, but in all of the the others it's still `mentioned`.
Personally I would prefer `reported`, but in any case, it should be consistent.
Same for making `duplicate` bold.

Reply:
Good point, I can make those changes later today.